packages/@react-aria/spinbutton/src/useSpinButton.ts

```diff
@@ -176,6 +176,7 @@ export function useSpinButton(
   return {
     spinButtonProps: {
       role: 'spinbutton',
+      tabIndex: isDisabled ? -1 : undefined,
```

Comment:
We can see here that a disabled spin button only has `aria-disabled`, not `disabled`

Is there a reason you chose to use tabIndex to solve this, rather than expand the definition of disabled in FocusScope?
https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/focus/src/FocusScope.tsx#L270

I'll bring it up with the team to see if we have any preferences.

Reply:
Ah no, we can definitely use `disabled` here instead.

Reply:
ah, no, i meant in FocusScope adding `[aria-disabled]` next to `[disabled]`

Reply:
Not sure whether I like using aria semantics to change focus order to be honest, feels wrong.

Reply:
Sorry, just looked at this again more closely.
The issue is just that we're too accepting of `contenteditable` https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/focus/src/FocusScope.tsx#L283
useDateSegment disabled has it set to false, so it's not a focusable element

We should restrict FocusScope to be any content editable EXCLUDING false (since contenteditable can be a couple different values https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/contenteditable#value)

but we should be able to do the exclude query selector
`[editable]:not([editable^="false"])`